docs/configure.md

```diff
@@ -393,67 +393,66 @@ See [the official documentation](https://kubernetes.io/docs/concepts/configurati
-   #
-   # For example, if you are using GKE with a cluster whose nodes are all in the "us-central1-a" zone, you could use the following values:
-   #
-   # provisioner: kubernetes.io/gce-pd
-   # parameters:
-   #  type: pd-ssd
-   #  zones: us-central1-a
-   ```
+### Google Cloud Platform (GCP)
 
-1. Read through the [Kubernetes storage class documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/), and fill in the `provisioner` and `parameters` fields in `base/sourcegraph.StorageClass.yaml` with the correct values for your hosting provider (e.x.: [GCP](https://kubernetes.io/docs/concepts/storage/storage-classes/#gce-pd), [AWS](https://kubernetes.io/docs/concepts/storage/storage-classes/#aws), [Azure](https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk)).
+```yaml
+# base/sourcegraph.StorageClass.yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: sourcegraph
+  labels:
+    deploy: sourcegraph
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-ssd
+```
 
-   - Note that if you're using GCP with Kubernetes `v1.9.*`, you should omit the `replication-type` parameter mentioned in [the documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#gce-pd) from your `base/sourcegraph.StorageClass.yaml` file. That field wasn't added until Kubernetes `v.1.10.*+`, and you'll see errors like the following if you try to use it with an older version:
+[Additional documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#gce-pd).
 
-     ```
-     Failed to provision volume with StorageClass "sourcegraph": invalid option "replication-type" for volume plugin kubernetes.io/gce-pd
-     ```
+### Amazon Web Services (AWS)
 
-   - **We highly recommend that the storage class use SSDs as the underlying disk type.** Using the snippets below will create a storage class backed by SSDs:
+```yaml
+# base/sourcegraph.StorageClass.yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: sourcegraph
+  labels:
+    deploy: sourcegraph
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+```
 
-     - [GCP](https://kubernetes.io/docs/concepts/storage/storage-classes/#gce-pd):
+[Additional documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs).
 
-       ```yaml
-       # base/sourcegraph.StorageClass.yaml
-       provisioner: kubernetes.io/gce-pd
-       parameters:
-         type: pd-ssd
-       ```
+### Azure
 
-     - [AWS](https://kubernetes.io/docs/concepts/storage/storage-classes/#aws):
+```yaml
+# base/sourcegraph.StorageClass.yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: sourcegraph
+  labels:
+    deploy: sourcegraph
```

Comment:
The Azure documentation notes that certain VMs might not be able to attach a premium LRS disk. This should be stated here as well.